kernel/packages/shared/world/parcelSceneManager.ts

```diff
@@ -46,14 +46,6 @@ export function stopParcelSceneWorker(worker: SceneWorker) {
   }
 }
 
-export function stopPortableExperienceWorker(cid: string) {
```

Comment:
I've moved this method to `dcl.ts`, is that expected?

Reply:
It could be a "portable experiences" special file/namespace